data/json/items/armor/helmets.json

```
@@ -629,7 +629,7 @@
     "material_thickness": 5,
     "environmental_protection": 1,
     "techniques": [ "WBLOCK_1" ],
-    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT" ]
+    "flags": [ "VARSIZE", "STURDY" ]
```

The rest of the changes are fine, but this was put here intentionally.
```suggestion
    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT" ]
```